### ISSUE TITLE
Allow plain strings as labels

### ISSUE THREAD

**Issue by keenancrane** [penrose/penrose#721]
**Note:** This issue is related to (but distinct from) #393, which addresses label strings in Style (whereas this issue is about Substance).

**Is your feature request related to a problem? Please describe.**
According to the [label docs here](https://github.com/penrose/penrose/wiki/Labels-in-Penrose), labels are currently specified via
```
Label x s
```
where `s` is a dollar-delimited LaTeX string.  E.g.,
```
Label p0 $p_0$
```
However, there's no great way to get plain-text strings (rather than math) as labels.

**Describe the solution you'd like**
Would be great to just be able to specify a plain-text string, delimited by (say) double quotes.  E.g.,
```
Label p0 "Planet Earth"
```
would yield a string that gets typeset as ordinary text.  Some kind of delimiter (like double quotes) seems necessary in order to allow strings to include whitespace.  Of course, there then needs to be a mechanism for escaping the delimiter, e.g., a backslash:
```
Label p0 "Planet \"Earth\""
```
Another possibility would be to just treat everything after the object name (and initial whitespace) as the label, with no delimiter, but that seems a bit hard to understand for the programmer.

**Describe alternatives you've considered**
  A kludge is to use the `\text{}` command from LaTeX, e.g.,
```
Label p0 $\text{Planet Earth}$
```
will typeset "Planet Earth" in upright characters, rather than in math-mode italics.  But this is obviously not a very user-friendly nor legible solution.

**Additional context**
To implement plain-text labels, we could just pass $\text{plain-text-string}$ to the current renderer.  I.e., we don't have to make "deep" changes to the system to get this to happen.

The other thing we'll have to think about is the possibility of including TeX strings inside plain-text labels.  E.g., is
```
Label p0 "Earth, Sector 001-$\alpha$"
```
allowed?  Or if someone wants to combine plain text and TeX, should they instead be required to write
```
Label p0 "\text{Earth, Sector 00-1}\alpha"
```
?  The former seems better, but is more work to implement.

**Comment by samestep** [comment]
> To implement plain-text labels, we could just pass $\text{plain-text-string}$ to the current renderer. I.e., we don't have to make "deep" changes to the system to get this to happen.
> 
> The other thing we'll have to think about is the possibility of including TeX strings inside plain-text labels. E.g., is
> 
> ```
> Label p0 "Earth, Sector 001-$\alpha$"
> ```
> 
> allowed? Or if someone wants to combine plain text and TeX, should they instead be required to write
> 
> ```
> Label p0 "\text{Earth, Sector 00-1}\alpha"
> ```
> 
> ? The former seems better, but is more work to implement.

Could you elaborate on why the former would be more work to implement? If we're translating `"plain-text-string"` to `$\text{plain-text-string}$` then `"Earth, Sector 001-$\alpha$"` would essentially get translated to `$\text{Earth, Sector 001-$\alpha$}$`, which should work fine, right?

**Comment by keenancrane** [comment]
> $\text{Earth, Sector 001-$\alpha$}$, which should work fine, right?

Oh... right. :-)

I simply forgot that TeX handles that case for you.

**Comment by keenancrane** [comment]
Some potentially useful discussion about different ways to integrate SVG and MathJax here: https://stackoverflow.com/questions/15962325/mathjax-inside-svg

A stopgap solution may be to have a `PlainText` shape that corresponds to SVG `\<text\>`, rather than text that gets typeset by MathJax.  This way we could also expose richer SVG text features, like stroking, etc.

(Ironically, the ability to export an SVG containing plain text is critical for using Penrose diagrams within ordinary LaTeX: for this we need to be able to get at the TeX formula as plain text, so that the local TeX engine can typeset it!)

**Comment by keenancrane** [comment]
Resolved by #740.
